text/0788-bedrock-agentcore-memory-l2.md

```diff
@@ -131,21 +131,21 @@ each optimized for specific use cases.
 For example: An agent helps multiple users with cloud storage setup. From these conversations,
 see how each strategy processes users expressing confusion about account connection:
 
-1. **Summarization Strategy** (`MemoryStrategy.usingBuiltInSummarization()`)
+1. **Summarization Strategy** (`MemoryStrategy.usingBuiltInSummarization(scope)`)
```

Comment:
No scope is needed here, the L2 of the memory construct would go through its builtin strategies and set their name this way. How it does that is an implementation detail, you don't need to specify how its done in the RFC.